samples/bluetooth/ipsp/README.rst

```diff
@@ -27,14 +27,132 @@ following commands:
    $ modprobe bluetooth_6lowpan
    $ echo 1 > /sys/kernel/debug/bluetooth/6lowpan_enable
 
-Then to connect:
+If you connected your board to a UART console, you will see an output similar to
+(may vary slightly by application and Zephyr versions):
+
+.. code-block:: console
+
+   [bt] [WRN] set_static_addr: Using temporary static random address
+   [bt] [INF] show_dev_info: Identity: cb:af:14:57:d8:6e (random)
+   [bt] [INF] show_dev_info: HCI: version 5.0 (0x09) revision 0x0000, manufacturer 0xffff
+   [bt] [INF] show_dev_info: LMP: version 5.0 (0x09) subver 0xffff
+   [bt] [WRN] bt_pub_key_gen: ECC HCI commands not available
+   [ipsp] [INF] init_app: Run IPSP sample
+   [ipsp] [INF] listen: Starting to wait
+
+The output above provides a BLE addressed assigned to your board (for the
+current session, will change on the next run).
+
+Alternatively, you may scan for your board on the host (this usually should
```

Comment:
If it needs to run as root, I'd recommend adding `sudo` to the command to emphasize that and delete the "this usually should be run as root".   From comments below, sounds like `hcitool` may not be the right one to use either.